tests/manual/test-procedure.rst

```diff
@@ -446,6 +446,23 @@ Test Case: Autocomplete
 - [ ] Disable autocomplete in "Options" and confirm that autocomplete no longer offers suggestions.
 
 
+Test Case A11y: Keyboard focus follows a logical order
+''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Using your keyboard, navigate through the menu and any submenus using the `tab` and `enter` or `space` keys.
+- [ ] Using your keyboard, navigate through the modal windows using the `tab` and `enter` or `space` keys.
```

Comment:
I think we need to be a bit more specific here, if we give this to somebody new to the project they might find it hard to identify issues if the expected behaviour is no clear.